labels/mapping.go

```diff
@@ -21,11 +47,11 @@ var core = []Label{
 
 var plugin = []Label{
 	{"Bug", "Something isn't working", "d73a4a"},
-	{"Duplicate", "This issue or pull request already exists", "cfd3d7"},
+	// {"Duplicate", "This issue or pull request already exists", "cfd3d7"},
 	{"Enhancement", "New feature or request", "a2eeef"},
 	{"Needs Mattermost Changes", "Requires changes to the Mattermost Plugin tookit", "9c14c9"},
 	{"Question", "Further information is requested", "d876e3"},
-	{"Wontfix", "This will not be worked on", "ffffff"},
+	// {"Wontfix", "This will not be worked on", "ffffff"},
```

Comment:
Same reason as above. I don't think a PR/issue could be closed without a label. This makes clear, why it got closed.

Reply:
could be closed without a label - not sure I agree, seems like it leads to more labels that we may not ever _query_ on. But, really 0/5. We can add them all for now, and then see if we actually use them.